ember_debug/view-debug.js

```diff
@@ -634,6 +634,20 @@ export default EmberObject.extend(PortMixin, {
   },
 
   /**
+   * Gather the children assigned to the render node.
+   *
+   * @param  {Object} renderNode
+   * @return {Array} children
+   */
+  _childrenForNode(renderNode) {
+    if (!!renderNode.morphMap) {
```

Comment:
This is causing the linting failure (and subsequently the CI failure). Changing to:

```javascript
if (renderNode.morphMap) { }
```

Should fix.